### ISSUE TITLE
Update demo example

### ISSUE THREAD

**Issue by mathiasrw** [agershun/alasql#270]
Some baras on http://alasql.org/demo/005d3/# looks "funny" when year is selected

**Comment by agershun** [comment]
Yes, this is not clean demo. We will fix it after finish with Codex or we will integrate all these demos with Codex.

**Comment by mathiasrw** [comment]
Is this integrated into codex now?
